embedchain/apps/llama2_app.py

```diff
@@ -8,9 +8,9 @@
 
 
 @register_deserializable
-class Llama2App(App):
+class llama2_app(App):
```

Comment:
This change is not needed. We want to follow the title casing for the class names.